deploy/olm-catalog/jaeger-operator/1.17.1/jaeger-operator.v1.17.1.clusterserviceversion.yaml

```diff
@@ -77,6 +77,7 @@ spec:
   customresourcedefinitions:
     owned:
     - description: Jaeger is the Schema for the jaegers API
+      displayName: Jaeger
```

Comment:
If this has to be done manually after every `generate` then it should be documented somewhere

Reply:
I don't know if there's an annotation missing somewhere, or if it's a bug in the CSV generator. Something to be investigated as part of another task.

Reply:
The issue is that we will probably forget to change this during the next release.

Reply:
Agree with you. I do suggest creating another task to investigate this, either to fix the problem or to document it in the release instructions.